tpu-provisioner/go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/GoogleCloudPlatform/ai-on-gke/tpu-provisioner
 
-go 1.22
+go 1.22.0
```

Comment:
Did you test building the image after this change? I had to remove the ".0" otherwise the image wouldn't build.

Reply:
Yes - retesting now though - PS: this was a change that was made by `go` itself

Reply:
Confirmed a `podman build` works for me